man/man1/arcstat.1

```diff
@@ -392,7 +392,17 @@ ARC reclaim needed
 \fBfree \fR
 .ad
 .RS 14n
-ARC free memory
+The ARC's idea of how much free memory there is, which includes evictable memory in the page cache.
```

Comment:
As we've discussed offline, I think we should either add more explanations on what this stat means exactly, or refer to `avail` as the stat that most developers would likely be interested in.

Reply:
Updated this to refer to `avail`.